app/src/main/java/de/grobox/transportr/favorites/trips/AbstractFavoritesPopupMenu.java

```diff
@@ -64,4 +71,22 @@ public boolean onMenuItemClick(MenuItem item) {
 		}
 	}
 
+	protected void addShortcut(Context context, Intent shortcutIntent, String shortcutName, int shortcutDrawable) {
```

Comment:
Do you really need to pass in a `Context` here or can you use the existing `context` variable?

The `shortcutDrawable` should be annotated with `@DrawableRes`.